pkg/proxy/iptables/proxier_test.go

```diff
@@ -2949,9 +3055,7 @@ func compareEndpointsMaps(t *testing.T, tci int, newMap proxy.EndpointsMap, expe
 					continue
 				}
 				if newEp.Endpoint != expected[x][i].Endpoint ||
-					newEp.IsLocal != expected[x][i].IsLocal ||
-					newEp.protocol != expected[x][i].protocol ||
```

Comment:
Just double checking - protocol is irrelevant here and can safely be removed right?

Reply:
Yes. It was actually always irrelevant.